src/interfaces/records-write.ts

```diff
@@ -140,7 +141,7 @@ export type CreateFromOptions = {
 /**
  * A class representing a RecordsWrite DWN message.
  */
-export class RecordsWrite {
+export class RecordsWrite implements MessageInterface<RecordsWriteMessage> {
```

Comment:
Sanity: Just want to make sure I understand this, we're not extending the new AbstractMessage here because this is a message that is potentially unsigned and returns the error when attempting to access?

Reply:
Mainly because of conflicting `_message` type is not just of `M` generic type.

Reply:
Thanks, makes sense! Added note helps for future reference.